pkg/ccl/streamingccl/streamingest/stream_ingestion_processor.go

```diff
@@ -181,6 +200,23 @@ func (sip *streamIngestionProcessor) Start(ctx context.Context) context.Context
 	}
 	sip.eventCh = sip.merge(sip.Ctx, eventChs)
 
+	// Start listening for signals from the poller processor.
+	go func() {
+		row, meta := sip.input.Next()
+		if meta != nil {
+			if meta.Err != nil {
+				sip.MoveToDraining(nil /* err */)
+			}
+			sip.ingestionMeta = meta
```

Comment:
i'm not a 100% sure if I'm handling this meta correctly. Should we just have another channel (like the signalCh) where we send the meta?